src/App.jsx

```diff
@@ -1,9 +1,12 @@
 import './App.css';
+import SignIn from './pages/SignIn';
+import SignUp from './pages/SignUp';
```

Comment:
Those pages should live in a file structure like this for example: pages/SignInPage/<filename>. Please alter that.